services/sentry/eventAggregator.js

```diff
@@ -51,10 +52,13 @@ function makeRecorder(channelId) {
 		leading: false,
 		trailing: true
 	})
-
 	const updateChannelPrice = async ev => {
-		const price = ev.type === 'UPDATE_IMPRESSION_PRICE' ? ev.price : '0'
-		await channelsCol.updateOne({ id: channelId }, { $set: { currentPricePerImpression: price } })
+		if (ev.type === eventTypes.IMPRESSION_PRICE_PER_CASE) {
```

Comment:
later improvement: consider merging `IMPRESSION_PRICE_PER_CASE` and `UPDATE_IMPRESSION_PRICE` into one event (considering that this is still not frozen in spec)

Reply:
Ohh okay, which event name should it be `UPDATE_IMPRESSION_PRICE `?

Reply:
Yes, but start with a PR to the documentation with the proposed interface

Reply:
see this: https://github.com/AdExNetwork/adex-validator/issues/203